plugins/sudo/sudo.plugin.sh

```diff
@@ -0,0 +1,14 @@
+#! bash oh-my-bash.module
+
+add_sudo() {
```

Comment:
```suggestion
function _omb_plugin_sudo_add_sudo {
```

- We are gradually switching to the naming convention with the prefix `_omb_plugin_{plugin_name}_` not to clutter the user's command/function space.
- We prefer `function fn` to `fn()` to avoid unexpected alias expansions.

Can you also update the function name in `bind -x`?

Reply:
fixed in last commit